app/Hooks/Consumers/Push.php

```diff
@@ -165,6 +172,11 @@ public function process(\PhpAmqpLib\Message\AMQPMessage $message)
 			if ( ! is_readable($repositoryName)) {
 				$this->logger->addNotice(sprintf('Repozitář "%s" nebyl na serveru nalezen', $repositoryName), $loggingContext);
 
+				if ($build) {
+					$this->logger->addNotice(sprintf('Do fronty bude zařazen nový pokus o sestavení repositáře "%s" pro větev "%s"', $repositoryName, $branchName), $loggingContext);
+					$this->createTestServerProducer->publish($build->id);
```

Comment:
Ahoj Milane @MilanPala ,
```suggestion
					$this->createTestServerProducer->publish($build->id);
					return self::MSG_ACK;
```
Nevím jestli bych neměl vracet MSG_ACK a ne až `self::MSG_REJECT;` (za podmínkou) 
Pokud je to stav, se kterým počítám a vlastně dál pokračuji, tak aby se nestalo, že se to vrátí třeba do DLX fronty a mezitím v druhé frontě už byl pokyn pro vytvoření repozitáře. 

Měl bych do těla podmínky po odeslání přidat   `return MSG_ACK;`?